notebooks/10_text_data.ipynb

```diff
@@ -0,0 +1,680 @@
+{
```

Comment:
<p>(or try out) yoursel -&gt; (or try out for yourself)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>